server/services/philips-hue/lib/light/light.getLights.js

```diff
@@ -29,52 +29,20 @@ async function getLights() {
     const lights = await hueApi.lights.getAll();
 
     lights.forEach((philipsHueLight) => {
-      switch (philipsHueLight.modelid) {
-        case 'LCT007': // hue white & color 2nd generation
-        case 'LCT015': // hue white & color 4th generation
-        case 'LST002': // hue lightstrip indoor 2nd generation
-        case 'LCT024': // Hue play 1
-        case '440400982841': // Hue White and Color Ambiance Play Light Bar
-        case 'LCT010': // Hue A19 White & Color w/ Richer Colors
-        case 'LCA001': // Hue color lamp
-        case 'GL-C-008': // Non-hue LED Strip
-        case 'CLA60 TW OSRAM': // CLA60 TW OSRAM
```

Comment:
According to the `lights.json` file, the "CLA60 TW OSRAM" is a `"type": "Color temperature light"`, meaning that with this PR, it'll get a `getPhilipsHueWhiteTemperatureLight` and before it had a `getPhilipsHueColorTemperatureLight`. Was it a mistake before or is it a problem here? (just checking one by one :) )

Reply:
Ok seems it was a mistake before: https://www.amazon.co.uk/Osram-Lightify-lamp-Tuneable-white/dp/B014OI12V4

Nice catch!